### PR TITLE
RHDEVDOCS-5794: Update example

### PR DESCRIPTION
**Pull request by nebelsprachen**
Version(s):

`gitops-docs` 

Cherrypick to:
`gitops-docs-1.10` `gitops-docs-1.11` `gitops-docs-1.12`

Issue:

* [RHDEVDOCS-5794](https://issues.redhat.com/browse/RHDEVDOCS-5794)

Link to docs preview:

* https://71945--docspreview.netlify.app/openshift-gitops/latest/understanding_openshift_gitops/gathering-gitops-diagnostic-information-for-support

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/about-must-gather.adoc [hunk 1]
```diff
@@ -23,12 +23,10 @@ Alternatively, you can collect specific information by running the command with
...
-+
+.Example command
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather \
-  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion}
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/must-gather-rhel8:v1.10.0
 ----
 
 * To collect the audit logs, use the `-- /usr/bin/gather_audit_logs` argument, as described in a following section.
```
Comment (eromanova97) [inline modules/about-must-gather.adoc:32]: Below is one more "For example:" that should be changed to ".Example command"

And after that is another "For example:", that one does not represent command though. It seems to be the list of the created pods. So I would change it to something like: ".Example pods" .

**Review by eromanova97 — COMMENTED** [review]
Looks good! I have just one more small suggestion. Thank you!

File: modules/about-must-gather.adoc (hunk 1 above)
Comment (eromanova97) [reply modules/about-must-gather.adoc:32]: Looks good to me, thank you!

**Review by skopacz1 — APPROVED** [review]
LGTM!

**Review by chengfang — APPROVED** [review]
LGTM

**Review by jldohmann — APPROVED** [review]
LGTM, nice work!

**Pull request merged**